pythoncalculator/subtract.py

```diff
@@ -0,0 +1,2 @@
+def subtract(x, y):
+    return y - y
```

Comment:
Thanks @yaoxiao1999, I spotted an error in the subtract function. Would you be able to fix this please

Reply:
I think this may be correct

```suggestion
    return x - y
```